homework-submission/react-week2/src/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import HyfButton from "./components/HyfButton";
+import HyfInput from "./components/HyfInput";
+import TodoList from './components/TodoList'
+import TodoItems from './components/TodoItems'
+import "./App.css";
+
+
+function App() {
+  const uniqid = require('uniqid')
+
+  const [toDoItems, setTodoItems] = useState([
+    {
+      name: "eat cheese"
+    },
+    {
+      name: "fight monsters"
+    },
+    {
+      name: "dont fall over"
+    }
+  ]);
```

Comment:
It would be more neat to keep track if a todo item is done or not here in the todo (sub)-objects. So that you have all the data in one place. The code of marking a todo item done would be easier to read that way.